testdata/test_types/types.go

```diff
@@ -0,0 +1,53 @@
+package test_types
```

Comment:
I'd rather see these packages outside of `testdata/`, which is supposed to contain static data/fixtures for the tests, not type definitions for the test code.

This could live in the `openapi/` folder, within subfolders.